dali/operators/image/resize/resize_attr.cc

```diff
@@ -213,45 +239,21 @@ void ResizeAttr::PrepareResizeParams(const OpSpec &spec, const ArgumentWorkspace
     }
 
     for (int i = 0; i < N; i++) {
-      for (int d = 0; d < spatial_ndim_; d++) {
-        requested_size[d] = size_vecs[d][i];
-      }
-
-      bool empty_input = volume(input_shape.tensor_shape_span(i)) == 0;
-      CalculateInputRoI(in_lo, in_hi, has_roi_, roi_relative_, make_cspan(roi_start_),
-                        make_cspan(roi_end_), input_shape, i, spatial_ndim_, first_spatial_dim_);
-      CalculateSampleParams(params_[i], requested_size, in_lo, in_hi, subpixel_scale_, empty_input,
-                            spatial_ndim_, mode_, max_size);
```

Comment:
I initially misunderstood how ResizeCropMirror works, so I made some customization points here. There was a lot of hastily written repetitive code which I refactored in the process. After discovering my mistake, I decided to keep those changes.